### ISSUE TITLE
Events throws NotSupportedException

### ISSUE THREAD

**Issue by persn** [zompinc/sync-method-generator#81]
Consider the following code

```cs
        public event EventHandler MyEvent;

        [Zomp.SyncMethodGenerator.CreateSyncVersion(OmitNullableDirective = true)]
        public async Task FooAsync()
        {
            MyEvent?.Invoke(this, EventArgs.Empty);
            await Task.Delay(100);
        }
```

It will fail to produce code with the following message `Generator 'SyncMethodSourceGenerator' failed to generate source. It will not contribute to the output and compilation errors may occur as a result. Exception was of type 'NotSupportedException' with message 'Can't process ClassLibrary1.Class18.MyEvent'.`

**Comment by virzak** [comment]
Will look into all your outstanding issues soon. Apologies for the delay.

**Comment by virzak** [comment]
This and other closed issues are resolved in 1.4.8

**Comment by persn** [comment]
It appears to be working, thank you 👍
